### PR TITLE
Add first draft of beta and prototype descriptions

### PR DESCRIPTION
**Pull request by Joelkw**
The goal of this page is to provide a place we can link to in-product or in emails and documentation to explicitly convey the expectations around our not-generally-available features. 

In addition to wording tweaks (welcome!), particularly want feedback on: 

- Do we want to use the word "prototype" or "experimental" for the "earlier than beta" stage (or change both to "alpha")? I think that there is a useful distinction in prototype vs beta (see the descriptions here) so my opinion is that we should keep something. Code insights uses prototype; [other places](https://docs.sourcegraph.com/admin/validation) use "experimental." 

- We don't have an official definition of either of these prior to this PR. Please do make comments/discuss in order to align this page with your expectations, current process, or what you think is best!

### CODE REVIEW

**Comment by Joelkw** [comment]
(Because this is entirely new and not just copied from some prior consensus, I will leave this PR open until everyone has had a chance to comment/review/re-review and approved it.)

**Comment by abschu** [comment]
@Joelkw - I proposed a few changes. Re: prototype vs experimental, I don't have a preference, but agree that we should be consistent.

File: doc/admin/beta_and_prototype_features.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Beta and prototype features
+
+Sourcegraph often beta-tests major features before we make them generally available. 
+
+Exposing beta and prototype features gives you an opportunity to try our newest features soonest, and for no additional cost.
```
Comment (Joelkw) [inline doc/admin/beta_and_prototype_features.md:5]:
```suggestion
Exposing beta and prototype features gives you an opportunity to try our newest features sooner.
```

File: doc/admin/beta_and_prototype_features.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Sourcegraph often beta-tests major features before we make them generally available. 
+
+Exposing beta and prototype features gives you an opportunity to try our newest features sooner.
+
+In return, your feedback helps us make sure that our new features are reliable and useful, and we appreciate any and all feedback you want to provide. You can [file an issue](https://github.com/sourcegraph/sourcegraph/issues/new/choose), email it directly to [feedback@sourcegraph.com](mailto:feedback@sourcegraph.com), or use the in-app feedback modal. 
+
+## **Both** beta and prototype features
+
+For both features labeled `Beta` and those labelled `Prototype`, Sourcegraph wants to be transparent that:
```
Comment (Joelkw) [inline doc/admin/beta_and_prototype_features.md:11]:
```suggestion
For both features labeled `Beta` and those labeled `Prototype`, Sourcegraph wants to be transparent that:  
```

File: doc/admin/beta_and_prototype_features.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Beta and prototype features
+
+Sourcegraph often beta-tests major features before we make them generally available.
```
Comment (Joelkw) [inline doc/admin/beta_and_prototype_features.md:3]:
```suggestion
Sourcegraph often beta tests major features before we make them generally available. 
```

**Comment by Joelkw** [comment]
@abschu did you have any more tweaks? I've incorporated all your changes in your original PR!

If not, can you mark this as "reviewed"? (Just trying to track the necessary reviews here.)

**Review by abschu — APPROVED** [review]
Looks good - thanks!

**Comment by Joelkw** [comment]
Forgot to remind folks in the product backlog meeting so doing it here – whenever you have a moment, would love a review from @nicksnyder @christinaforney @macraig @malomarrec @rrhyne !

Blocking until all reviewers have seen, since there are customer commitments in here (like "prototypes will never be enabled by default, betas can be;" "all betas have docs") that affect all teams and that you might have a strong opinion on.

**Review by rrhyne — APPROVED** [review]
LGTM!

**Comment by malomarrec** [comment]
LGTM!
Note to self - we'll have to stop using "Experimental"

**Review by malomarrec — APPROVED** [review]

**Comment by Joelkw** [comment]
@malomarrec I don't have a strong preference between "prototype" and "experimental." Do you? Open to either! 

cc @abschu did you have an opinion on this question? To me, they both convey "early stage," "subject to change," "likely to break," "for the purposes of learning" etc.

**Review by macraig — APPROVED** [review]
Thanks for adding @Joelkw  and sorry for the delay!

**Review by nicksnyder — APPROVED** [review]
great, thanks!

**Pull request merged**